test/functional/lua/ftplugin_spec.lua

```diff
@@ -0,0 +1,105 @@
+local t = require('test.testutil')
+local n = require('test.functional.testnvim')()
+
+local exec_lua = n.exec_lua
+local eq = t.eq
+local root = n.exec_lua('return vim.fs.normalize(vim.fn.getcwd())')
+
+before_each(function()
+  n.command [[e `=stdpath('config') .. '/lua/user-foo/init.lua'`]]
+  n.command('w ++p')
+  n.command [[e `=stdpath('config') .. '/lua/user-foo/module-bar.lua'`]]
+  n.command('w ++p')
+
+  n.command [[e `=stdpath('data') .. '/site/pack/plugin/start/test-plugin/lua/pack-foo/init.lua'`]]
+  n.command('w ++p')
+  n.command [[e `=stdpath('data') .. '/site/pack/plugin/start/test-plugin/lua/pack-foo/module-bar.lua'`]]
+  n.command('w ++p')
+  n.clear()
+end)
+
+local function stdpath(type)
+  return exec_lua(string.format([[ return vim.fs.normalize(vim.fn.stdpath('%s')) ]], type))
+end
+
+local function vimruntime()
+  return exec_lua [[return vim.fs.normalize(vim.env.VIMRUNTIME)]]
+end
+
+describe('lua.includeexpr', function()
```

Comment:
option names are usually single-quoted by convention (or `&foo`)

```suggestion
describe("Lua 'includeexpr'", function()
```